packages/wallets/src/features/cfd/modals/MT5PasswordModal/MT5PasswordModalFooters.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { ComponentProps } from 'react';
+import { useHistory } from 'react-router-dom';
+import { WalletButton, WalletButtonGroup } from '../../../../components';
+import { useModal } from '../../../../components/ModalProvider';
+
+type TProps = {
+    disabled: ComponentProps<typeof WalletButton>['disabled'];
+    isDemo?: boolean;
+    isLoading: ComponentProps<typeof WalletButton>['isLoading'];
+    onPrimaryClick: ComponentProps<typeof WalletButton>['onClick'];
+    onSecondaryClick: ComponentProps<typeof WalletButton>['onClick'];
+};
+
+export const SuccessModalFooter = ({ isDemo }: Pick<TProps, 'isDemo'>) => {
+    const history = useHistory();
+    const { hide } = useModal();
+
+    if (isDemo) {
+        return (
+            <div className='wallets-success-btn'>
+                <WalletButton isFullWidth onClick={hide} size='lg'>
+                    OK
+                </WalletButton>
+            </div>
+        );
+    }
+
+    return (
+        <WalletButtonGroup isFlex isFullWidth>
+            <WalletButton onClick={hide} size='lg' variant='outlined'>
```

Comment:
`hide` doesn't sound like function 🤣

Reply:
`const { show, hide } = useModal();`

It's coming from this hook that was created to help in Modal handling.